File: backend/Server/Server/Models/Order.cs
using System;
using System.Collections.Generic;

namespace Server.Models;

public class Order
{
    public int Id { get; set; }
    public DateTime CreatedAt { get; set; }
    //public decimal Total { get; set; }
    public int PaymentTypeId { get; set; }
    public PaymentsType PaymentsType { get; set; }

    public int UserId { get; set; }
    public User User { get; set; }
    public int WishlistId { get; set; }
    public Wishlist Wishlist { get; set; }

    public string HashOrSession { get; set; }

    public long Total { get; set; }
    public decimal TotalETH { get; set; } 


}

File: frontend/src/app/models/order.ts
import { Product } from "./product";
import { Wishlist } from "./wishlist";

export class Order {
    id: number;
    createdAt: Date;
    paymentTypeId: number;
    userId : number;
    wishlist: Wishlist;
    total : number;
    totalETH : number;


    constructor(id: number, createdAt: Date, PaymentTypeId : number , UserId: number, Wishlist: Wishlist,
        total : number, totalETH : number)
    {
        this.id = id,
        this.createdAt = createdAt,
        this.paymentTypeId = PaymentTypeId,
        this.userId = UserId
        this.wishlist = Wishlist;
        this.total = total;
        this.totalETH = totalETH;
        
    }
}
